pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/OpChainExecutionContext.java

```diff
@@ -102,4 +106,8 @@ public OpChainStats getStats() {
   public boolean isTraceEnabled() {
     return _traceEnabled;
   }
+
+  public void yield() {
```

Comment:
not sure if this is not a good place for yield method. we can revisit this

Reply:
this is the only place i can think of that allows ANY operator to yield from their own position --> this is the goal of this PR --> to avoid returning no-op blocks and requiring all operators to handle positional yielding back to opChain root.